### PR TITLE
narrow the promtail k8s pod discovery to kube-system only 

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement

**What this PR does / why we need it**:
With this PR when the `promtail` agent installed on each shoot node is trying to discover a pod, it narrows it only to the `kube-system` namespace.
Also the request and limits of the loki's `kube-RBAC-proxy` are increased to meet the increased load.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
Logs from shoot components can be retrieved only from pods running in the shoot `kube-system` namespace.
```

### CODE REVIEW

File: charts/seed-bootstrap/charts/loki/values.yaml [hunk 1]
```diff
@@ -65,11 +65,11 @@ resources:
...
     limits:
-      cpu: 60m
-      memory: 50Mi
+      cpu: 180m
+      memory: 150Mi
     requests:
-      cpu: 30m
-      memory: 15Mi 
+      cpu: 50m
+      memory: 50Mi
```
Comment (rfranzke) [inline charts/seed-bootstrap/charts/loki/values.yaml:72]: cc @stoyanr

**Review by rfranzke — APPROVED** [review]
/lgtm

**Comment by vpnachev** [comment]
I had to check that https://github.com/gardener/gardener/pull/5600 is available only in `>=v1.43`

**Pull request merged**